src/Linux/local_cache.cpp

```diff
@@ -268,6 +273,18 @@ static std::string get_file_name(const std::string& id)
     return g_cache_dirname + "/" + sha256(id);
 }
 
+std::string get_cached_file_location(const std::string& id){
+	return get_file_name(id);
+}
+
+std::string get_last_cache_read_expiry(){
+	std::string result("Last cache read expiry is unset. This could be because there's been no cache reads yet, it has been read without checking for cache expiry or an error during cache read before cache expiry is checked.");

```

Comment:
Is this in the right place? With this positioning no matter what this log is going to be sent even if the cache read expiry is set.